### ISSUE TITLE
Problen with gquest detection

### ISSUE THREAD

**Issue by AardNaricain** [AardCrowley/Search-and-Destroy#39]
Joined a gquest, this happened:
```
gq join
You have now joined Global Quest # 1. See 'help gquest' for available commands.
gq check

[7023/7023hp 6842/7060mn 4780/4780mv 0qt 550tnl] >
You notice the grey moon falling to the west.
main_target_list build error - tried to run while on gq but area/room type is 'none'.
   No target items to show.
[7023/7023hp 7024/7060mn 4780/4780mv 0qt 550tnl] >
You still have to kill 1 * a withered valkyrie (Black Lagoon)
You still have to kill 1 * Suljina the Nix (Rebellion of the Nix)
You still have to kill 1 * river rapids (Tanra'vea)
You still have to kill 1 * a blue light (Tanra'vea)
You still have to kill 2 * a feeding demon (The Cataclysm)
You still have to kill 1 * Drilla (The Covenant of Mistridge)
You still have to kill 1 * a Necrophim (The Covenant of Mistridge)
You still have to kill 2 * a lava leech (The Deadlights)
You still have to kill 1 * Destiny (The Partroxis)
You still have to kill 1 * Olivia (The Partroxis)
You still have to kill 1 * Daestitrus (The Partroxis)
```

No targets showing up in the target window. When I did `gq_info` it sorted itself out
```
---------------------[  CURRENT GLOBAL QUEST DETAILS ]---------------------
Quest Name.........: [ Global quest # 1 ]
Quest Type ........: [ Main Group ]
Quest Status.......: [ Extended ]
Finished Time .....: [ Sun Jun 06 15:51:09 ]
Level range........: [  186 ] - [  201 ]
Completed by ......: [ Areia ]
Extended Time .....: [ 00:01:55 ]
Kills in Extra Time: [ 17 ]
Completed in Extra : [ 1 ]
Inactivity Timer...: [ 00:00:11 ]
-----------------------------[ Quest Rewards ]-----------------------------
Quest points.......: [    13 ] ( +3 for each mob killed )
Trivia points......: [     1 ]
Gold Coins.........: [ 18600 ]
-----------------------------[ Quest Targets ]-----------------------------
To complete this quest, you must:
Kill at least 1 * a withered valkyrie (Black Lagoon).
Kill at least 1 * Suljina the Nix (Rebellion of the Nix).
Kill at least 1 * river rapids (Tanra'vea).
Kill at least 1 * a blue light (Tanra'vea).
Kill at least 2 * a feeding demon (The Cataclysm).
Kill at least 1 * Drilla (The Covenant of Mistridge).
Kill at least 1 * a Necrophim (The Covenant of Mistridge).
Kill at least 2 * a lava leech (The Deadlights).
Kill at least 1 * Destiny (The Partroxis).
Kill at least 1 * Olivia (The Partroxis).
Kill at least 1 * Daestitrus (The Partroxis).
---------------------------------------------------------------------------
gq type detection: area (level 201)

[7023/7023hp 7024/7060mn 4780/4780mv 0qt 550tnl] >
--------------------------------------------------------------------------------
 1  a withered valkyrie              - lagoon      [notes]                      
 2  Suljina the Nix                  - rebellion   [notes]                      
 3  river rapids                     - tanra       [notes]                      
 4  a blue light                     - tanra       [notes]                      
 5  a feeding demon                  - cataclysm   [notes]                      
 6  Drilla                           - mistridge   [notes]                      
 7  a Necrophim                      - mistridge   [notes]                      
 8  a lava leech                     - deadlights  [notes]                      
 9  Destiny                          - partroxis   [notes]                      
10  Olivia                           - partroxis   [notes]                      
11  Daestitrus                       - partroxis   [notes]                      
--------------------------------------------------------------------------------
Type 'xcp <index>' or click link to go to that target.
```

**Comment by AardNaricain** [comment]
gquest info when not in a gquest:
`There is more than one global quest running.`

gq check when not in a global quest:
`You are not in a global quest.`

**Comment by AardNaricain** [comment]
Should be fixed by #51
